docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -398,6 +398,23 @@ public class CustomConfiguration {
 }
 ----
 
+==== End User attributes
+
+When enabled, Quarkus adds OpenTelemetry End User attributes as Span attributes.
```

Comment:
We should mention which security libraries must be present on order for this to work.